### DISCUSSION TITLE
WHAT TO DO AS AN AP Graduate in Computer Science?!

### DISCUSSION THREAD

**Discussion by EnQiridion** [community/community#118714] [New to GitHub]
Question:

After the summer 2024, i am starting as an AP Graduate in Computer Science. Right now i am practicing at FreeCodeCamp to prepare myself. I am curious though on what you think the best free coding practice site is out there. and if you can recommend any good languages for a beginner that know a little HTML and very little JAVA.

**Comment by ettaboyle** [comment]
Thanks for posting in the GitHub Community, @EnQiridion!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. **The Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions).** 

This question should be in the [New to GitHub](https://github.com/orgs/community/discussions/categories/new-to-github) category. I've gone ahead and moved it for you. Good luck!

**Comment by ghostinhershell** [comment]
Hey there @EnQiridion, 

I'm sure that answers will vary here but I got my start by earning the Responsive Web Design certification from FreeCodeCamp! 

We've also got some community posts that may be useful for you to take a look at. I'll link them here:
- [Getting Started with Programming Languages Hub 🗣️](https://github.com/orgs/community/discussions/51394)
- [You can really do anything you want.. with transferable skills!](https://github.com/orgs/community/discussions/109396)

Good luck! 🍀
